package.json

```diff
@@ -98,17 +95,11 @@
     "electron-devtools-installer": "^3.2.0",
     "electron-packager": "^14.0.6",
     "electron-rebuild": "^3.2.7",
-    "eslint": "^5.16.0",
-    "eslint-plugin-import": "^2.18.2",
-    "eslint-plugin-node": "^9.2.0",
-    "eslint-plugin-promise": "^4.2.1",
-    "eslint-plugin-react": "^7.14.3",
-    "eslint-plugin-standard": "^4.0.1",
```

Comment:
These conflicted with dependencies from react-scripts v5.  Could we just accept the config and conventions from react-scripts?

Reply:
It looks like `eslint` has been removed completely as a dev dependency, but there's still an `.eslintrc`. Might upgrading to the newest `eslint` and plugins help?

Reply:
We do still have eslint and plugins as dev dependencies, but we're getting them as the opinionated defaults from react-scripts instead of choosing our own.  So the `.eslintrc.js` is still being used, and I think it's working OK now.